assets/js/qa.js

```diff
@@ -0,0 +1,47 @@
+var currentQuestion = 0;
+var score = 0;
+var totQuestions = questions.length;
+
+
+var container =document.getElementById("quizContainer");
+var questionEl = document.getElementById("question");
+var opt1 = document.getElementById("opt1");
+var opt2 = document.getElementById("opt2");
+var opt3 = document.getElementById("opt3");
+var opt4 = document.getElementById("opt4");
+var nextButton = document.getElementById("nextButton");
+var resultCont = document.getElementById("result");
+
+
+function loadQuestion (questionIndex) {
+    var q = questions[questionIndex];
+    questionEl.textContent = (questionIndex + 1) + '.' + q.question;
+    opt1.textContent = q.option1;
+    opt2.textContent = q.option2;
+    opt3.textContent = q.option3;
+    opt4.textContent = q.option4;
+};
+
+function loadNextQuestion () {
+    var selectedOption = document.querySelector('input[type=radio]:checked');
+    if(!selectedOption){
+        alert('please select your answer!');
+        return;
+    }
+    var answer = selectedOption.Value;
```

Comment:
The word `value` should not have a capital `V`. It returns `undefined` at the moment, breaking the answer check:

```suggestion
    var answer = selectedOption.value;
```